pkg/comp-functions/runtime/desired.go

```diff
@@ -69,8 +69,24 @@ func (d *DesiredResources) GetFromObject(ctx context.Context, o client.Object, k
 	return d.fromKubeObject(ctx, ko, o)
 }
 
-// PutIntoObject adds or updates the desired resource into its kube object
+// PutIntoObject adds or updates the desired resource into its kube object, from docs:
+//
+// The provider can fully manage the resource.
 func (d *DesiredResources) PutIntoObject(ctx context.Context, o client.Object, kon string, refs ...xkube.Reference) error {
+	return d.putIntoObject(ctx, false, o, kon, refs...)
+}
+
```

Comment:
```suggestion
// PutIntoObserveOnlyObject adds or updates the desired resource into its OBSERVE-ONLY kube object, from docs:
```

Or it won't be a valid godoc line.